lib/src/main/java/com/transloadit/android/sdk/Transloadit.java

```diff
@@ -0,0 +1,39 @@
+package com.transloadit.android.sdk;
+
+import android.app.Activity;
+
+import com.transloadit.sdk.async.AssemblyProgressListener;
+
+import org.jetbrains.annotations.Nullable;
+
+
+public class Transloadit extends com.transloadit.sdk.Transloadit {
```

Comment:
Can this class be renamed to AndroidTransloadit to highlight the difference to com.transloadit.sdk.Transloadit?